Source/WebCore/PAL/pal/crypto/commoncrypto/CryptoDigestCommonCrypto.cpp

```diff
@@ -129,9 +125,10 @@ static std::variant<std::unique_ptr<CC_SHA1_CTX>, std::unique_ptr<CC_SHA256_CTX>
         return WTF::makeUniqueWithoutFastMallocCheck<PAL::Digest>(PAL::Digest::sha512Init());
     }
 #endif
-    case CryptoDigest::Algorithm::SHA_224: {
+    case CryptoDigest::Algorithm::DEPRECATED_SHA_224: {
+        RELEASE_ASSERT_NOT_REACHED_WITH_MESSAGE("SHA224 is not supported.");
         std::unique_ptr<CC_SHA256_CTX> context = WTF::makeUniqueWithoutFastMallocCheck<CC_SHA256_CTX>();
-        CC_SHA224_Init(context.get());
+        CC_SHA256_Init(context.get());
         return context;
```

Comment:
None of the code after the `RELEASE_ASSERT()` is reachable.  We should just delete it, or does the compiler complain?

Reply:
I have kept it for the compiler.